exercise_perseus_render/assets/src/vue/attemptprogress.vue

```diff
@@ -9,9 +9,9 @@
       </div>
     </div>
 
-    <p id="message" v-if="passRatioN === passNum">{{ $tr("get") }} <b>{{passRatioN}} {{ $tr("outof") }} {{passRatioM}}</b> {{ $tr("correct") }}</p>
-    <p id="message" v-if="passNum > 0 && passRatioN !== passNum">{{ $tr("get") }} <b>{{ passNum }}</b> {{ $tr("more") }}</p>
-    <p id="message" v-if="passNum <= 0 && passRatioN !== passNum">{{ $tr("hooray") }}</p>
+    <p id="message" v-if="passRatioM === passNum">{{ $tr("get") }} <b>{{passRatioM}} {{ $tr("outof") }} {{passRatioN}}</b> {{ $tr("correct") }}</p>
+    <p id="message" v-if="passNum > 0 && passRatioM !== passNum">{{ $tr("get") }} <b>{{ passNum }}</b> {{ $tr("more") }}</p>
+    <p id="message" v-if="passNum <= 0 && passRatioM !== passNum">{{ $tr("hooray") }}</p>
```

Comment:
this certainly not the ideal solution, but since they can put any string around the 2 numbers, they have the flexibility of doing something like "{get} 2 {out of} 3 {correct}" or "{the goal is get} 2 {correct, there are in total} 3 {questions}"